contrib/completion/bash/docker

```diff
@@ -334,14 +334,26 @@ _docker_diff() {
 
 _docker_events() {
 	case "$prev" in
-		--since)
+		--filter|-f)
+			COMPREPLY=( $( compgen -S = -W "container event image" -- "$cur" ) )
+			compopt -o nospace
+			return
+			;;
+		--since|--until)
+			return
+			;;
+	esac
+
+	case "${words[$cword-2]}$prev=" in
+		*event=*)
+			COMPREPLY=( $( compgen -W "create destroy die export kill pause restart start stop unpause" -- "${cur#=}" ) )
```

Comment:
With `:` as a separator it would definitely be simpler because `:` does not get parsed into an element of `words`. 
I don't understand why this is so, both `:` and `=` are part of `COMP_WORDBREAKS`.

Anyway, thanks for reviewing! I'll continue this way for the remaining completions.